project-base/tests/App/Acceptance/acceptance/PageObject/Front/LoginPage.php

```diff
@@ -4,19 +4,30 @@
 
 namespace Tests\App\Acceptance\acceptance\PageObject\Front;
 
+use Facebook\WebDriver\WebDriverBy;
 use Tests\App\Acceptance\acceptance\PageObject\AbstractPage;
 
 class LoginPage extends AbstractPage
 {
-    /**
-     * @param string $email
-     * @param string $password
-     */
-    public function login($email, $password)
+    protected const DEFAULT_USER_NAME = 'Jaromír Jágr';
+    protected const DEFAULT_USER_EMAIL = 'no-reply@shopsys.com';
+    protected const DEFAULT_USER_PASSWORD = 'user123';
+
+    public function login()
     {
-        $this->tester->fillFieldByName('front_login_form[email]', $email);
-        $this->tester->fillFieldByName('front_login_form[password]', $password);
-        $this->tester->clickByName('front_login_form[login]');
+        $emailInput = $this->webDriver->findElement(WebDriverBy::name('front_login_form[email]'));
```

Comment:
Can this call be extracted into `findElementByName` method? 

Also, is it more reliable to find element first and then fillByElement? Can you explain to me why is that?

Reply:
I have changed this behavior.